config/locales/translation_ja.yml

```
@@ -9,3 +9,10 @@ ja:
         memo: メモ  #g
         picture: 画像  #g
         title: 題名  #g
+
+  pagination:
+    first: '最初'
+    last: '最後'
+    previous: '前'
+    next: '次'
+    truncate: '...'
```

これでも間違いではないのですが、HTML上は3点リーダ記号をあらわす文字実体参照があるので、そちらを使ってもらうほうが省略の意味が伝わりやすいかと思います。
```suggestion
    truncate: '&hellip;'
```
https://en.wikipedia.org/wiki/Ellipsis
https://qiita.com/ktr1211/items/115a9eb037496e3c047d
https://e-words.jp/w/%E4%B8%89%E7%82%B9%E3%83%AA%E3%83%BC%E3%83%80%E3%83%BC.html